priv/repo/migrations/20170504190323_create_tasks.exs

```diff
@@ -0,0 +1,24 @@
+defmodule CoursePlanner.Repo.Migrations.CreateTasks do
+  use Ecto.Migration
+
+  def change do
+    execute("""
+          CREATE TYPE task_status AS ENUM (
+            'Pending',
+            'Accomplished'
+          )
+    """)
+    create table(:tasks) do
+      add :name, :string
+      add :deadline, :date
+      add :status, :task_status
```

Comment:
I replied to the same somewhere else. status is not important cause tracking volunteers is way more simple, by just seeing whether they do it or not. i think we can drop the status